### PR TITLE
[NETBEANS-3702 : Fixed hint 'implements all abstract method' for record interfaces]

### PR DESCRIPTION
**Pull request by singh-akhilesh**
https://issues.apache.org/jira/browse/NETBEANS-3702

### CODE REVIEW

File: java/java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java [hunk 1]
```diff
@@ -3972,7 +3977,7 @@ else if (Kind.VARIABLE == tree.getKind()) {
         }
         return result;
     }
-
+
```
Comment (ebarboni) [inline java/java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java]: Hi can you remove this space added change

Comment (singh-akhilesh) [reply java/java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java]: Indeed..

Comment (singh-akhilesh) [reply java/java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java]: comments addressed

**Comment by ebarboni** [comment]
is it possible to rebase on your side ? I'm not sure squash and base works.

**Comment by singh-akhilesh** [comment]
> is it possible to rebase on your side ? I'm not sure squash and base works.

**Comment by singh-akhilesh** [comment]
> > is it possible to rebase on your side ? I'm not sure squash and base works.

Done

**Comment by arusinha** [comment]
👍 , I had verified the fix . It is working fine.

**Comment by ebarboni** [comment]
thks for rebase effort and @arusinha for review

**Pull request merged**